VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -171,4 +166,10 @@ protected void hideMessageInSitu(){
         }
     }
 
+    private void showErrorDialog(String message) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setMessage(message)
+                .setPositiveButton(R.string.label_ok, null)
+                .create().show();
```

Comment:
@bguertin: What's the scenario in which the Activity may be destroyed while the dialog is still being displayed?